etc/ghostwriter.profile

```diff
@@ -0,0 +1,50 @@
+# Firejail profile for ghostwriter
+# Description: Cross-platform, aesthetic, distraction-free Markdown editor.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include ghostwriter.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/ghostwriter
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+
+mkdir ${HOME}/.config/ghostwriter
+whitelist ${HOME}/.config/ghostwriter
+include whitelist-common.inc
```

Comment:
If whitelists are included then markdown files from Documents and other locations can't be accessed.

Reply:
Uhh good argument.
Is it better to whitelist ${Documents}, ${Downaloads}, ... or remove the whitelist?

Reply:
For existing text editors we allow everywhere
just include disable-xdg and noblacklist documents

Reply:
I think adding ${Documents}, ${Downloads} would be enough.

Reply:
let me reword that
```
include disable-common.inc
include disable-passwdmgr.inc
include disable-programs.inc
```
are in all the current editors, and should be included here too

Reply:
My idea:
```
noblacklist ${HOME}/.config/ghostwriter
noblacklist ${DOCUMENTS}
noblacklist ${PICTURES}

include disable-common.inc
include disable-devel.inc
include disable-interpreters.inc
include disable-passwdmgr.inc
include disable-programs.inc
include disable-xdg.inc

mkdir ${HOME}/.config/ghostwriter
mkdir ${DESKTOP}
mkdir ${DOCUMENTS}
mkdir ${DOWNLOADS}
mkdir ${PICTURES}
whitelist ${HOME}/.config/ghostwriter
whitelist ${DESKTOP}
whitelist ${DOCUMENTS}
whitelist ${DOWNLOADS}
whitelist ${PICTURES}
include whitelist-common.inc
```

Reply:
@SkewedZeppelin if you remove whitelist then all (non explicitly blacklisted) dotfiles will be accessible. Are we expecting users will store markdown files somewhere in dotfiles?

@rusty-snake why ${PICTURES}? The rest looks good to me.

Reply:
@Vincent43 For inserting screen photos, for example.

Reply:
For this I use `*.local` in `~/.config/firejail.profile`

Reply:
Yeah, this is double edged sword. Certainly some users wouldn't want that every random app can access their custom home dirs and dotfiles. IMO it's better to fail-safe here rather than fail-open and use `*.local` for overrides.

Reply:
Now the whitelit is comment out.